internal/venice-test-common/build.gradle

```diff
@@ -323,6 +323,48 @@ task generateGHCI() {
   Files.move(targetFilePath, finalDestinationPath, StandardCopyOption.REPLACE_EXISTING)
 }
 
+task generateGHCIMaster() {
```

Comment:
Can we keep just one task to generate all the GH workflows?

(Internally, we can have multiple tasks, but the primary one should depend on those tasks to generate all the workflows that our repo needs)